src/garage/experiment/local_runner.py

```diff
@@ -165,39 +178,47 @@ def restore(self, from_dir, from_epoch='last'):
                 Not applicable when snapshot_mode='last'.
 
         Returns:
-            A SimpleNamespace for train()'s arguments.
+            types.SimpleNamespace: A SimpleNamespace for train()'s arguments.
```

Comment:
i think it is starting to become time to define `class TrainArgs:` --- this return type doesn't tell me anything about what to expect. with `TrainArgs` I can just look at the type.